common/src/main/java/com/joyent/http/signature/crypto/NssBridgeKeyConverter.java

```diff
@@ -31,35 +25,6 @@
  * @see <a href="http://docs.oracle.com/javase/8/docs/technotes/guides/security/p11guide.html">PKCS#11 Reference Guide</a>
  * @see <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS">Network Security Services</a>
  */
+@Deprecated
```

Comment:
Why `Deprecated` and delete the code?  Wouldn't we typically do one or the other?

Reply:
The class is still public, removing it without prior deprecation seems too much of a change to make for a single release.